### PR TITLE
Added some captures about private items.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Expanded information related to private item #1204

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Expanded info for issue #1204

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/permissions.md [hunk 1]
```diff
@@ -52,6 +52,14 @@ The following permissions **are** supported:
 
 - **Private items**: When you grant **Full Access** permission to a mailbox, you can decide whether to allow the delegate to see private items (private meetings, appointments, contacts, or tasks) in the mailbox.
 
+> [!IMPORTANT]
+> The Private items can be shared with a delegated using the following procedure on your outlook desktop app:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/permissions.md]:
```suggestion
> The Private items can be shared with a delegate using the following procedure on your Outlook on Desktop app:
```

File: Exchange/ExchangeHybrid/permissions.md [hunk 2]
```diff
@@ -52,6 +52,14 @@ The following permissions **are** supported:
...
+> 1. Go to File/Account Settings/Delegate Access
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/permissions.md]:
```suggestion
> 1. Go to **File** > **Account Settings** > **Delegate Access**
```

File: Exchange/ExchangeHybrid/permissions.md [hunk 3]
```diff
@@ -52,6 +52,14 @@ The following permissions **are** supported:
...
+>  ![Delegate Access Menu 1](../../README-media/Private_Item_Menu1.png)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/permissions.md]:
```suggestion
>  ![Delegate Access Menu 1](../../README-media/Private_Item_Menu1.png).
```

File: Exchange/ExchangeHybrid/permissions.md [hunk 4]
```diff
@@ -52,6 +52,14 @@ The following permissions **are** supported:
...
+> 2. On the next windows click on Add. A new menu will appears with the people inside of your organization, select a delegate and then click Ok.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/permissions.md]:
```suggestion
> 2. On the next, windows click on **Add**. A new menu will appear listing the people in your organization. Select a delegate and click **OK**.
```

File: Exchange/ExchangeHybrid/permissions.md [hunk 5]
```diff
@@ -52,6 +52,14 @@ The following permissions **are** supported:
...
+> 3. The following Image will appears, where you can select the Checkbox related share private items with a delegate
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/permissions.md]:
```suggestion
> 3. The following image will appear, where you can select the related checkbox to share private items with a delegate.
```

**Comment by j0rt3g4** [comment]
@scanum  Done Manuela

**Comment by chrisda** [comment]
A couple of issues:

1. The screenshots you added were MASSIVE. Use a maximum of 600 wide, and preferable send them through tinypng.com.

2. The pictures belong in the media folder (in this case ExchangeHybrid\media), and you can't upload them there. Just make your changes as if the images are in the correct media folder and attach them to the PR

**Pull request merged**